src/components/Requirements/SubRequirement.vue

```diff
@@ -305,9 +304,38 @@ export default Vue.extend({
         ? `${this.subReq.minCountFulfilled}/${this.subReq.minCountRequired} ${this.subReq.fulfilledBy}`
         : 'self check';
     },
-    fulfilledSelfCheckCourses(): readonly FirestoreSemesterCourse[] {
-      const reqId = this.subReq.requirement.id;
-      return store.state.derivedSelectableRequirementData.requirementToCoursesMap[reqId];
+    fulfilledSelfCheckCourses(): readonly CourseTaken[] {
```

Comment:
Some comments in this function would be helpful! Like about `selectedCourses` v `fulfilledCourses`. Why we filter `fulfilledCourses` for just AP/IB, etc.